html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/data-url-agent-cluster.https.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/wasm/jsapi/wasm-module-builder.js"></script>
+<div id=log></div>
+<script>
+const dataURLWorker = `
+  onmessage = () => self.postMessage("FAIL");
+  onmessageerror = () => self.postMessage("PASS");
+`;
+
+const dataFrameScript = `
+  onmessage = () => parent.postMessage("FAIL", "*");
+  onmessageerror = () => parent.postMessage("FAIL", "*");
+`;
+
+[
+  new SharedArrayBuffer(),
+  new WebAssembly.Module(new WasmModuleBuilder().toBuffer())
```

Comment:
Should we test WebAssembly.Memory too?

Reply:
That follows the underlying object (a literal AB or SAB) so I don't think it's needed, but we could. If you also wanted to test it backed by AB though we'd have to invert all the pass conditions for that variant.